tests/Routing/RoutingRouteTest.php

```diff
@@ -526,6 +526,7 @@ public function testLeadingParamDoesntReceiveForwardSlashOnEmptyPath()
 
     /**
      * @expectedException \Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+     * @expectedExceptionMessage
```

Comment:
This doesn't do anything, since the expected message only checks it starts with what you write, so if you are trying to test there is no message, this won't work.

Reply:
Good point,
I will change it